components/Bookmark/index.js

```diff
@@ -1,38 +1,24 @@
-import React, { useState } from "react";
+import React from "react";
 import Image from "next/image";
 import styled from "styled-components";
 
-export default function Bookmark() {
-  const [bookmarked, setBookmarked] = useState(false);
-
+export default function Bookmark({ handleBookmark, item, isBookmarked }) {
   function handleClick() {
```

Comment:
I would maybe rename the function to `handleBookmarkClick` when you refactor the code so it is more clear for other devs 🙂